packages/core/src/Stores/contract-store.js

```diff
@@ -433,6 +484,15 @@ function calculate_marker(contract_info, { accu_high_barrier, accu_low_barrier }
     if (tick_count >= 1) {
         if (!isDigitContract(contract_type)) {
             // TickContract
+            if (isResetContract(contract_type)) {
+                return {
+                    contract_info: toJS(contract_info),
+                    type: 'TickContract',
+                    key: `${contract_id}-date_start`,
+                    epoch_array: [reset_time, ...ticks_epoch_array],
+                    price_array: [price_array[0], reset_barrier],
```

Comment:
`price_array[0]` from `[price_array[0], reset_barrier]` is a price of initial (default) barrier